src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt

```diff
@@ -97,6 +101,22 @@ class SolImportPathReference(element: SolImportPathElement) : SolReferenceBase<S
       }
     }
 
+    private fun remappingsFromFoundryConfigFile(file: VirtualFile): List<Pair<String, String>> {
+      val foundryConfigFile = file.findFileByRelativePath("foundry.toml")
+      val remappings = arrayListOf<Pair<String, String>>()
+      if (foundryConfigFile != null) {
+        val mapper = TomlMapper()
+        val data = mapper.readValue(foundryConfigFile.readText(), Map::class.java)
+        (((data["profile"] as LinkedHashMap<*, *>)["default"] as LinkedHashMap<*, *>)["remappings"] as ArrayList<*>).forEach { mapping ->
```

Comment:
Would we need to be more defensive here with accessing the data? I can imagine the case where someone is editing the file and some of the nested blocks are missing. I can imagine it'd possibly produce an error because `null` could be casted to `LinkedHashMap<*, *>`?